System/Vector.cpp

```diff
@@ -60,30 +60,20 @@ namespace RTE {
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-	Vector & Vector::RadRotate(float angle) {
-		angle = -angle;
-		const float tempX = m_X * std::cos(angle) - m_Y * std::sin(angle);
-		const float tempY = m_X * std::sin(angle) + m_Y * std::cos(angle);
-		m_X = tempX;
-		m_Y = tempY;
+	Vector & Vector::RadRotate(const float angle) {
+		const float adjustedAngle = -angle;
```

Comment:
As far as I know, there's no reason to const local variables like this. No need to change it really (except consistency I guess), but I wouldn't advise bothering with it in future.

My take on it is that compilers are smart enough to figure it out, so there's no efficiency benefit. So its only effect is that the variable can't be modified by someone fiddling with the code, but I'd say that's pretty pointless - I think people will either be smart enough not to modify stuff that shouldn't be modified, or they'll just remove the const on it cause it's local to the function so they don't care.

Google the subject if you want other people's thoughts, and I'm happy to discuss in discord of course.

Reply:
Yeah, it's not for efficiency, it's for readability and safety mainly.
"Const everything" is advice that I've ran into a lot lately which is why I'm starting to do it.
Specifically here I agree that it was pedantic of me to write const in such a small scope.

Reply:
Yeah, I can absolutely see and argument for it with complex methods or, of course, reference parameters, but I don't think it makes a ton of sense in small things like these.